specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1094,16 +1097,19 @@
         "accessMode": {
           "type": "string",
           "enum": [
-            "Unspecified",
-            "EnforceMode",
-            "LearningMode",
-            "DryRunMode"
+            "Learning",
+            "Enforced",
+            "Audit"
           ],
           "x-ms-enum": {
             "name": "AssociationAccessMode",
             "modelAsString": true
           },
           "description": "Access mode on the association."
+        },
+        "hasProvisioningIssues": {
+          "type": "string",
+          "description": "Has provisioning issues"
```

Comment:
Specifies if there are any provisioning issues.

Reply:
Please model this as an enum so that only "yes" and "no" are allowed

Reply:
Discussed offline: enum not required. Changed description.